spec/draft-barnes-cfrg-4q.md

```diff
@@ -317,28 +318,34 @@ valid representation of a point.
 Not all 32 byte strings represent valid points. Implementations MUST
 reject invalid strings and check that decompression is
 successful. Strings are invalid if they are not possible outputs of
-the compression operator. In particular the values of y0 and y1 MUST
+the compression operator.  In particular the values of y0 and y1 MUST
 be less then p.
 
 # Scalar multiplication
 
-We now present two algorithms for scalar multiplication on the above curve.
-Both use the same addition and doubling formulas, and one is a simple windowed
-exponentiation, while the other uses endomorphisms to accelerate computation.
+Below, we present two algorithms for scalar multiplication on the above curve: 
+one uses a simple fixed-window exponentiation without exploiting endomorphisms,
+while the other uses endomorphisms to accelerate computation.  
+The execution of operations in both algorithms has a regular pattern in order 
+to facilitate protection against timing and simple side channel attacks.  Both 
+algorithms use the same addition and doubling formulas.
+
+First, we discuss explicit formulas and efficient projective coordinate
+representations.
 
 ## Alternative Point Representations and Addition Laws
 
 We use coordinates based on extended twisted Edwards coordinates
 introduced in {{TwistedRevisited}}: the tuple (X, Y, Z, T) with Z
 nonzero and Z \* T = X \* Y corresponds to a point (x, y) satisfying x
-= X/Z and y = Y/Z. The point at infinity in this representation is
+= X/Z and y = Y/Z. The neutral point in this representation is
 (0,1,1,0). The following slight variants are used in the optimized
 scalar multiplication algorithm in order to save computations: point
 representation R1 is given by (X,Y,Z,Ta,Tb), where T=Ta\*Tb;
 representation R2 is (N, D, E, F) = (X+Y,Y-X,2Z,2dT); representation
 R3 is (N, D, Z, T) = (X+Y,Y-X,Z,T); and representation R4 is
-(X,Y,Z). R2 representation was introduced in {{Ed25519}} to accelerate
-repeated additions of the same point. Converting between these
+(X,Y,Z). Similar "caching" techniques were discussed in {{TwistedRevisited}} 
+to accelerate repeated additions of the same point. Converting between these
```

Comment:
It might be good to specify these conversions in an appendix sometime in the future, but we don't need to do that now.